pkg/kube/client.go

```diff
@@ -428,40 +428,39 @@ func updateResource(c *Client, target *resource.Info, currentObj runtime.Object,
 		if err := target.Get(); err != nil {
 			return fmt.Errorf("error trying to refresh resource information: %v", err)
 		}
-		return nil
-	}
-
-	// send patch to server
-	helper := resource.NewHelper(target.Client, target.Mapping)
-
-	obj, err := helper.Patch(target.Namespace, target.Name, patchType, patch)
-	if err != nil {
-		kind := target.Mapping.GroupVersionKind.Kind
-		log.Printf("Cannot patch %s: %q (%v)", kind, target.Name, err)
+	} else {
+		// send patch to server
+		helper := resource.NewHelper(target.Client, target.Mapping)
 
-		if force {
-			// Attempt to delete...
-			if err := deleteResource(c, target); err != nil {
+		obj, err := helper.Patch(target.Namespace, target.Name, patchType, patch)
+		if err != nil {
+			kind := target.Mapping.GroupVersionKind.Kind
+			log.Printf("Cannot patch %s: %q (%v)", kind, target.Name, err)
+
+			if force {
```

Comment:
I would reverse this condition so that the code is more simple (less indented):

```
if !force {
    return err
}

// your force code here
```

Reply:
That's existing code I did not touch.